pr-data.csv

```diff
@@ -2178,10 +2178,10 @@ https://github.com/cloudera/altus-sdk-java,914688f557923a8d9fae44b4da318e3e66c96
 https://github.com/CloudSlang/cloud-slang,48923570780dd096e0cf17ef657b6b1fdc50dc31,cloudslang-entities,io.cloudslang.fortest.SensitiveValueTest.testEncryptedStringSensitiveValue,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/79
 https://github.com/CloudSlang/cloud-slang,48923570780dd096e0cf17ef657b6b1fdc50dc31,cloudslang-entities,io.cloudslang.fortest.SensitiveValueTest.testEncryptedStringSensitiveValuePreEncrypted,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/79
 https://github.com/CloudSlang/cloud-slang,48923570780dd096e0cf17ef657b6b1fdc50dc31,cloudslang-entities,io.cloudslang.fortest.SensitiveValueTest.testSensitiveValueEncryptDecrypt,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/79
-https://github.com/CloudSlang/cloud-slang,76b733019b362f551481f1bdc78ed2eac73b1bd4,cloudslang-compiler,io.cloudslang.lang.compiler.MetadataExtractorTest.testMetadataPrettyPrint,ID,Opened,https://github.com/CloudSlang/cloud-slang/pull/1384,
-https://github.com/CloudSlang/cloud-slang,76b733019b362f551481f1bdc78ed2eac73b1bd4,cloudslang-compiler,io.cloudslang.lang.compiler.MetadataHelperTest.testPrettyPrint,ID,Opened,https://github.com/CloudSlang/cloud-slang/pull/1384,
-https://github.com/CloudSlang/cloud-slang,76b733019b362f551481f1bdc78ed2eac73b1bd4,cloudslang-compiler,io.cloudslang.lang.compiler.modeller.transformers.SeqStepsTransformerTest.testTransformStepWithEmptyReqKeys,ID,Opened,https://github.com/CloudSlang/cloud-slang/pull/1383,
-https://github.com/CloudSlang/cloud-slang,76b733019b362f551481f1bdc78ed2eac73b1bd4,cloudslang-compiler,io.cloudslang.lang.compiler.modeller.transformers.SeqStepsTransformerTest.testTransformStepWithMissingReqKeys,ID,Opened,https://github.com/CloudSlang/cloud-slang/pull/1383,
+https://github.com/CloudSlang/cloud-slang,76b733019b362f551481f1bdc78ed2eac73b1bd4,cloudslang-compiler,io.cloudslang.lang.compiler.MetadataExtractorTest.testMetadataPrettyPrint,ID,Accepted,https://github.com/CloudSlang/cloud-slang/pull/1384,
```

Comment:
The change is approved, but the PR has not been merged. Leave it as `Opened` for now